app/jobs/clock/subscriptions_to_be_terminated_job.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Clock
+  class SubscriptionsToBeTerminatedJob < ApplicationJob
+    queue_as 'clock'
+
+    def perform
+      Subscription
+        .joins(customer: :organization)
+        .joins('left join webhooks on subscriptions.id = webhooks.object_id and '\
+               "webhooks.webhook_type = 'alert.subscription_to_be_terminated' and webhooks.status = 1")
```

Comment:
Since we have an auto-retry system on the webhooks delivery. Should we really check the status?
The risk here is that we send twice or more the same webhook if the first attempt was not a success. 

My point is that we should let the webhook engine responsible for ensuring the right delivery of the webhook, and try to push only one alert at this step. What do you think?

Reply:
make sense 👍